.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+---
+
+## 🐛 Bug
+
+<!-- A clear and concise description of what the bug is. -->
+
+### To Reproduce
+
+<!-- Minimal code example to reproduce the bug. -->
+
+### Expected behavior
+
+<!-- FILL IN -->
+
+### Environment
+
+- Pyodide Version (e.g. 1.8.1):
+- Browser version (e.g. Chrome 95.0.4638.54):
+- Any other relevant information:
```

Comment:
I think examples should be in comments, otherwise people will leave them in and it will add noise to the bug report,

```suggestion
- Pyodide version: <!-- e.g. 1.8.1 -->
- Browser version: <!-- e.g. Chrome 95.0.4638.54 -->
- Any other relevant information:
```